tox_docker.py

```diff
@@ -100,6 +100,7 @@ def getint(reader, key):
             "healthcheck_timeout": gettime(reader, "healthcheck_timeout"),
             "healthcheck_retries": getint(reader, "healthcheck_retries"),
             "healthcheck_start_period": gettime(reader, "healthcheck_start_period"),
+            "ports": reader.getstring("ports"),
```

Comment:
Might save some code below to use https://github.com/tox-dev/tox/blob/fcba59158f853c4da38f164dbf8523b1aeb0fedf/src/tox/config/__init__.py#L1452 instead; this also lets users specify in the typical tox.ini list fashion (1 per line)

probably worth validating that ports are integers, too